app/frontend/stores/requirement-block-store.ts

```diff
@@ -123,6 +123,10 @@ export const RequirementBlockStoreModel = types
       }
       return response.ok
     }),
+    setRequirementBlockFilters(queryParams) {
+      // Set filters here once necessary
+      // See permit-application-store.ts for example
+    },
```

Comment:
idea the create searchModel should handle this if possible. Just make the filters param optional in the create function and escape filter related logic based on that. As not all searches will have filters

Reply:
I'm not sure I understand, it does handle this (in the same way it handles fethData using fetchDataActionName)

Reply:
@bkmaibach I am saying this shouldn't be required as not all searches use filtering